packages/network-controller/src/NetworkController.ts

```diff
@@ -63,32 +73,178 @@ export type NetworkMetadata = {
 };
 
 /**
- * Custom RPC network information
+ * The type of an RPC endpoint.
+ *
+ * @see CustomRpcEndpoint
+ * @see InfuraRpcEndpoint
+ */
```

Comment:
nit:
```suggestion
 * @see {@link CustomRpcEndpoint}
 * @see {@link InfuraRpcEndpoint}
 */
```

Reply:
Fixed in [a600e92](https://github.com/MetaMask/core/pull/4286/commits/a600e9244be9f77c88e9369f6aab7c833b0374a1).